byoeb-v1/byoeb/byoeb/services/chat/message_handlers/user_flow_handlers/generate.py

```
@@ -602,17 +602,21 @@ async def agenerate_answer(
+                return response, response
+
             patterns = {
                 "response_en": r"<response_en\s*>(.*?)</response_en\s*>",
                 "response_src": r"<response_src\s*>(.*?)</response_src\s*>",
             }
-
             extracted_data = {}
             for key, pattern in patterns.items():
                 match = re.search(pattern, xml_string, re.DOTALL | re.IGNORECASE)  # Supports multiline and case-insensitive matches
-                extracted_data[key] = match.group(1).strip() if match else None  # Strip removes extra spaces and newlines
+                if match is None or not match.group(1).strip(): raise ValueError("Parsing failed, did not match pattern '%s' (%s) in output" % (pattern, key))
```

Ruff (0.15.10)

[error] Multiple statements on one line (colon)

(E701)